<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRUtilityWidget</class>
 <widget class="QWidget" name="QRUtilityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Generator</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Scanner</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelQR">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditQR">
     <property name="placeholderText">
      <string>Enter text here to generate QR Code from it</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRWidget" name="widgetQR" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="labelQRScanner">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRWidget</class>
   <extends>QWidget</extends>
   <header location="global">qr/QRWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditQR</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QRUtilityWidget</receiver>
   <slot>qrTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>715</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>qrTextChanged(QString)</slot>
 </slots>
</ui>
